src/content/self-hosted/manage/troubleshooting.mdx

```diff
@@ -5,6 +5,45 @@ sidebar_label: Troubleshooting
 id: troubleshooting
 ---
 
+## How to extract logs from Okteto when asking for help
+
+When asking for assistance from Okteto Support, please use the [`okteto logs` command](reference/okteto-cli.mdx#logs) to fetch the logs from your Development Environment.
+
+## Understanding the state of your app: why endpoints aren't present in the CLI or UI
+
+Endpoint links are not present within Okteto if there are no services actively running. If your endpoints are missing, consider the following states and their implications:
+
+#### Progressing: 
+* Description: Your deployment is in the process of being rolled out
+* Possible Causes:
+  * Your service is still being started
+  * There are pending updates or new deployments
+* Actions:
+  * Wait for the deployment to complete
+  * Look at the events of the deployment for any issues
+
+#### Running:
+* Description: Your deployment is active, and the service should be running correctly
+* Possible Causes:
+  * The service is up, but there might be a configuration issue
+  * The annotation for automatic ingress might be missing
+  * Network policies or firewall rules could be blocking the endpoint
+* Actions:
+  * Verify the service annotations in your manifest to ensure `dev.okteto.com/auto-ingress: "true"` is present
+  * Inspect network policies to confirm they allow traffic to your service
```

Comment:
I would remove this one and add links to public endpoints for docker compose too

Reply:
Updated. Could you take a look?